### ISSUE TITLE
Python 3 pop-beagle does not plot the MAP SED

### ISSUE THREAD

**Issue by lukasfurtak** [jacopo-chevallard/PyP-BEAGLE#47]
Hi,

I switched to the python 3 version of pyp-beagle and when I run:

`pyp_beagle -r BEAGLE_outputs/Fit_all_spec-z --plot-marginal --show-MAP-SED`

it plots the photometry and the marginal photometry but not the best-fitting SED (see attached plot). Note the same command works fine in python 2.7 version pyp-beagle where it also plots the MAP SED as a black line.

[7338_BEAGLE_marginal_SED_phot.pdf](https://github.com/jacopo-chevallard/PyP-BEAGLE/files/8310882/7338_BEAGLE_marginal_SED_phot.pdf)

**Comment by jacopo-chevallard** [comment]
@ImaanBontle you recently found the issue, am I right ?

**Comment by ImaanBontle** [comment]
@lukasfurtak @jacopo-chevallard Yes, I have the same issue. PyP-BEAGLE seems to accept the command but not produce any visible output from it. I haven't found a solution so far.

**Comment by jacopo-chevallard** [comment]
@ImaanBontle @lukasfurtak it should be solved with version 0.9.3, you can update your version with `pip install pyp-beagle -U`

**Comment by ImaanBontle** [comment]
@jacopo-chevallard Seems fixed now!

**Comment by lukasfurtak** [comment]
@jacopo-chevallard Yes, works now, thanks!
